bindings/python/tests/test_python_bindings.py

```diff
@@ -32,9 +32,10 @@ def __repr__(self):
 
 
 def test(verbosity=1, stream=sys.stdout):
-    tests = [test for test in glob(os.path.join(thisdir, '*.py'))
-             if not test.endswith('__.py') and
-             not test.endswith('test_python_bindings.py')]
+    tests = [test for test in sorted(glob(os.path.join(thisdir, 'test_*.py')))
+             if not test.endswith('__.py')
+             and not test.endswith('test_python_bindings.py')
+             and not test.endswith('test_global_dlite_state.py')]
```

Comment:
Why do you exclude test_global_dlite_state.py?

Reply:
Because the test would be run in two different contexts and the global dlite state (istore) is then different making test assertions awkward. When run in test_python_bindings.py, previous tests have already populated the istore in ways which were not easy to understand (Because the other tests do not care about what they might populate the istore with, which is OK)

Reply:
Good point. Lets add a comment about that
```suggestion
             # Exclude test_global_dlite_state.py since the global state
             # that it is testing depends on the other tests.
             and not test.endswith('test_global_dlite_state.py')]
```